tests/lib/prepare.sh

```diff
@@ -44,6 +44,34 @@ EOF
     systemctl daemon-reload
 }
 
+# It is supposed the name of the system follows 
+# ubuntu-core-<VERSION>[-ARCH]-<BITS>
+# with arch is "" for amd64, arm for armhf and arm64, etc
+is_core() {
```

Comment:
maybe these should be named `is_test_target_core` so as to not give a false impression that the current system is UC.